### PR TITLE
Nazar/feature/rtl tr hebrew

### PR DESCRIPTION
**Pull request by Nazar-kob**

### CODE REVIEW

File: frontend/src/components/СustomModal/CustomModal.tsx [hunk 1]
```diff
@@ -1,3 +1,5 @@
+import { t } from "i18next";
```
Comment (Taraslysak) [inline frontend/src/components/СustomModal/CustomModal.tsx:1]: why do we need this import?

File: frontend/src/pages/ChangePassword/ChangePassword.tsx [hunk 2]
```diff
@@ -47,14 +41,14 @@ const ChangePassword = () => {
...
-        notify(textDataButtons[settingsViewKey.NOTIFY_MES]);
+        notify(t("changePassword.toastMessage"));
         navigate("/settings");
       }
     },
     onError: async (err) => {
       setError("password", {
         type: "bad password",
-        message: "Please provide a valid password.",
+        message: t("changePassword.errorPassword") as string,
```
Comment (Taraslysak) [inline frontend/src/pages/ChangePassword/ChangePassword.tsx:51]: why do we need these hard casting? can `t` return something other than string?

File: frontend/src/pages/Home/components/Orders/Orders.module.css [hunk 3]
```diff
@@ -24,16 +24,24 @@
...
     display: flex;
     flex-direction: column;
     overflow-y: scroll;
-    margin-right: 3.75rem;
-    margin-left: 0;
+    margin-inline-start: 3.75rem;
+    margin-inline-end: 0;
+
+    /* margin-inline-start: 3.75rem; */
+    /* margin-inline-end: 0; */
```
Comment (Taraslysak) [inline frontend/src/pages/Home/components/Orders/Orders.module.css:45]: redundant comments

File: frontend/src/pages/Home/components/Orders/Orders.tsx [hunk 4]
```diff
@@ -56,11 +58,11 @@ const Orders = ({ filterOptions, isArchive }: OrdersProps) => {
 
     if (activeBtnFilterName && sortFunction) {
       dataForSort.sort(sortFunction);
-      sortByActiveOrderId(dataForSort, arrayActiveOrders);
+      // sortByActiveOrderId(dataForSort, arrayActiveOrders);
       setOrdersData(dataForSort);
```
Comment (Taraslysak) [inline frontend/src/pages/Home/components/Orders/Orders.tsx:62]: redundant logs

File: frontend/src/pages/Home/components/Orders/Orders.tsx [hunk 5]
```diff
@@ -56,11 +58,11 @@ const Orders = ({ filterOptions, isArchive }: OrdersProps) => {
...
       return true;
     }
-    sortByActiveOrderId(dataForSort, arrayActiveOrders);
+    // sortByActiveOrderId(dataForSort, arrayActiveOrders);
```
Comment (Taraslysak) [inline frontend/src/pages/Home/components/Orders/Orders.tsx:65]: Redundant logs

File: frontend/src/pages/Market/Market.tsx [hunk 6]
```diff
@@ -53,15 +55,15 @@ export function Market() {
...
+  //   const timer = setTimeout(() => {
+  //     if (orderState.isNumberVerified) {
+  //       handlerStepNextClient();
+  //     }
+  //   }, 20000);
 
-    return () => clearTimeout(timer);
-  }, [orderState.isNumberVerified]);
+  //   return () => clearTimeout(timer);
+  // }, [orderState.isNumberVerified]);
```
Comment (Taraslysak) [inline frontend/src/pages/Market/Market.tsx:66]: Redundant comments

**Review by Taraslysak — COMMENTED** [review]
Good job!

File: frontend/src/pages/ChangePassword/ChangePassword.tsx (hunk 2 above)
Comment (Nazar-kob) [reply frontend/src/pages/ChangePassword/ChangePassword.tsx:51]: it shows the can be returned undefined

**Review by kipod — APPROVED** [review]

**Pull request merged**